cmd/root.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func newRootCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "sd-local",
+		Short: "Enable to build in local",
```

Comment:
How about the message like `Run build in local`. I think `Enable to~` is not suitable for the description of the command.

Reply:
I fixed it